Gemfile.jruby-3.1.lock.release

```diff
@@ -2,7 +2,7 @@ PATH
   remote: logstash-core-plugin-api
   specs:
     logstash-core-plugin-api (2.1.16-java)
-      logstash-core (= 8.16.5)
+      logstash-core (= 8.16.6)
```

Comment:
Please also update (cannot add a suggestion on not changed lines) L10 of this file.